### DISCUSSION TITLE
Initial Feedback

### DISCUSSION THREAD

**Discussion by chrisjsewell** [executablebooks/unified-myst#1] [General]
A place for any initial feedback

**Comment by rowanc1** [comment]
Curious to see how you incorporate the types from myst:
https://executablebooks.github.io/myst-spec/myst.schema.html#target

The spec types are published as myst-spec on npm. Looks like you are not setting the identifier (just the label) in this code?

**Reply by chrisjsewell** [reply]
See: https://github.com/executablebooks/unified-myst/blob/main/packages/target-extension/README.md#todo 😉

Yeh you can import types in JSDoc, that's how it works with all the micromark typing: https://github.com/executablebooks/unified-myst/blob/e11053e18963ee1a6d11ffb643ef052cbd486af6/packages/target-extension/src/lib/tokenize.js#L2
